src/main/java/beam/physsim/jdeqsim/cacc/sim/JDEQSimulation.java

```diff
@@ -52,7 +50,7 @@ protected void initializeVehicles() {
                 isCaccEnabled = value;
             }
             // the vehicle registers itself to the scheduler
-            new Vehicle(getScheduler(), person, activityDurationInterpretation, isCaccEnabled);
+            new Vehicle(getScheduler(), person, activityDurationInterpretation, isCaccEnabled, allRoads);
```

Comment:
Is this coming from JDEQSimulation? Is this part of the new matsim changes - can you link to that for review also? I know we try to not modify matsim too much to avoid merge issues later down the road

Reply:
That's correct. Though without this change (removing static fields) we cannot run tests in the same jvm. I had to change matsim. Though these changes aren't so big.